diff.py

```diff
@@ -2045,7 +2045,7 @@ def split_off_address(line: str) -> Tuple[str, str]:
         if len(parts) < 2:
             parts.append("")
     off = len(line) - len(parts[-1])
-    return line[:off], line[off:]
+    return line[:off], line[off:].strip()
```

Comment:
Can we do `off = len(line) - len(parts[-1].strip())` instead, to maintain the property that concatenating the resulting pair gives the original string back?

Reply:
Done